core/src/test/scala/io/projectglow/vcf/VCFDatasourceSuite.scala

```diff
@@ -693,6 +738,33 @@ class VCFDatasourceSuite extends GlowBaseTest {
         s"$testDataHome/vcf/missing_contig_length.vcf",
         s"$testDataHome/vcf/missing_contig_length.vcf")
   }
+
+  test("prune genotype fields") {
+    import sess.implicits._
+    val extraHeaderLines =
+      """
+        |##FORMAT=<ID=MIN_DP,Number=1,Type=Integer,Description="">
+        |##FORMAT=<ID=AD,Number=R,Type=Integer,Description="">
+        |""".stripMargin.trim() + "\n"
+    val df = parseVcfContents(
+      "chr0\t0\t.\tA\t.\t.\t.\t.\tAD:MIN_DP\t2,5:1",
```

Comment:
Nor immediately clear to me why `makeVcfLine` is not used like other tests and instead the first portion of the file  is provided directly.